WeCantSpell.Hunspell/WordList.Query.cs

```diff
@@ -466,7 +446,7 @@ private void SetSuffixAppend(string append)
 
                     do // simplified checkcompoundpattern loop
                     {
-                        if (CompoundCheckTimeLimiter is null)
+                        if (opLimiter.QueryForCancellation())
```

Comment:
To match the previous behavior, this should not perform a full `QueryForCancellation` but should just check the cached property instead. The query should occur above near line 426. Maybe this is better behavior though 🤷‍♂️.

Reply:
I'm going to keep it like this for better responsiveness to cancellation